lib/bcoin.js

```diff
@@ -92,19 +105,53 @@ class LedgerBcoin {
 
     const indexes = path;
     const data = await this.ledger.getPublicKey(path, addressFlags);
-    const rawPubkey = data.publicKey;
-    const compressedPubkey = secp256k1.publicKeyConvert(rawPubkey, true);
+    const compressed = secp256k1.publicKeyConvert(data.publicKey, true);
+
+    let pfp = 0;
+
+    if (parentFingerPrint)
+      pfp = await  this.getParentFingerprint(indexes);
 
     return new HDPublicKey({
```

Comment:
Is it possible to also return the `fingerprint` so that a parent and child `HDPublicKey` can be connected?

Reply:
It is returned with the HDPublicKey, if you use `getPublicKey().xubkey(net)` now you will get compatible xpubs.

Reply:
It is returned with the HDPublicKey, if you use `getPublicKey().xubkey(net)` now you will get compatible xpub strings.

Reply:
Ok awesome. Can you turn the function into `getFingerprint` instead of `getParentFingerprint` so that I can use it to get the master public key fingerprint?